packages/next/server/config-shared.ts

```diff
@@ -455,6 +456,7 @@ export const defaultConfig: NextConfig = {
     workerThreads: false,
     pageEnv: false,
     optimizeCss: false,
+    optimizeScripts: false,
```

Comment:
Do we want to call this something more specific to workers e.g. `nextScriptWorkers`?

Reply:
My initial plan was to have a single generic flag that would eventually control multiple experimental `next/script` features in the future, but maybe that just adds unnecessary confusion?

Also you're not the first person to suggest making it more specific; I just updated it to `nextScriptWorkers` :)